CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+## 0.5.0 (2018-06-04)
+
+### Breaking changes
+
+Changed returning value of all selection utils to `{ node, start, pos}`, where
+  * `start` points to the start position of the node
+  * `pos` points directly before the node
+  * `node` ProseMirror node
```

Comment:
You may wish to highlight that the previous value of `pos` used to point to the start position of the node, and now points to directly before the node to match ProseMirrors convention

Reply:
yeah, I thought I had done that, will do.